_posts/2018-06-23-kimai-v2-announcement.md

```diff
@@ -0,0 +1,71 @@
+title: "Announcement - Kimai v2"
+date: "2018-06-23 22:00:00 +0200"
+author: kevinpapst
+---
+
+Welcome to a long overdue blog posting with an announcement, which some of you didn't believe it will ever come ;-)
+
+But here it is ...
+
+## Today I am thrilled to announce that we are officially working on a brand new Kimai version
+
+Before I start with some background information, here is a list of links about Kimai v2:
+
+- You can find the new website for now at [https://v2.kimai.org](https://v2.kimai.org)
+- The project is developed at [at Github](https://github.com/kevinpapst/kimai2/)
+- We have a [demo site](https://v2.kimai.org/demo/) that is waiting for your tests
+- After testing, please leave us your [feedback](https://github.com/kevinpapst/kimai2/issues/) at GitHub, 
+so we can shape the future development roadmap together
+
+## Some background information
+
+Simon and I discussed about writing a new Kimai version for a long time and it was also 
+[requested by the community for many years](https://github.com/kimai/kimai/issues/259).
+
+We were always unsure whether the effort would pay out. Writing a brand new timetracking software in a modern framework would 
+basically mean writing a new software which would only share the same name, the same developer and probably the same approach 
+of tracking and managing timesheets.
+
+But there was a point when it became clear that Kimai v1 was overhauled and barely manageable any longer. 
+Writing new features for Kimai v1 felt wrong and painful, as the main codebase is more than 10 years old and less than ideal 
+considering the great toolkits the PHP community has to offer in 2018.
+
+Also Kimai v1 was never meant to be used on modern devices and even though there were third-party apps, 
+the support for mobile usage was not ideal - to say at least.
+
+So it was finally time "to let go" and start working on the next generation of Kimai and inspirational as developers are: 
+we call it "Kimai v2" for now ;-)
+
+I choose to use Symfony 4.1 as technical core layer and a bootstrap based theme, to be up-to-date with the latest 
+and greatest features from the open-source world and to be ready for another 10 years of Kimai.
+
+You can find a lot of technical detail information [at Github](https://github.com/kevinpapst/kimai2/) and in 
+the [developer documentation](https://github.com/kevinpapst/kimai2/blob/master/var/docs/developers.md).
+
+## Current status
+
+The status of the development is early and a lot of the advanced features from Kimai v1 are still missing.
+
+Its already used in production by myself to find all the small quirks in order to improve the everyday usage, 
+especially on mobile devices.
+
+Installation process is done in some minutes (you need PHP 7.1.3 or higher) and we already added support for 
+importing your data from Kimai v1. 
+
+But we will very likely not call it "production ready" until the end of the year, as we have a lot of improvements in the pipeline 
```

Comment:
ok. great. Then we should also publish our release cycle or something like that somewhere so that people know when they can expect a new release. We should not just randomly create new releasese like until now.
Second, we should talk about a business plan. Let's include our friend into this topic who is willing to support us